portal/server/controllers/utilisation-report-service/record-correction/record-correction-history/helpers.ts

```diff
@@ -0,0 +1,41 @@
+import { getKeyToDateSortValueMap, IsoDateTimeStamp } from '@ukef/dtfs2-common';
+import { format, parseISO } from 'date-fns';
+import { SHORT_FORM_DATE } from '../../../../constants/date';
+import { CompletedCorrectionViewModel } from '../../../../types/view-models/record-correction/record-correction-history';
+import { GetCompletedFeeRecordCorrectionsResponseBody } from '../../../../api-response-types';
+
+/**
+ * Formats the date sent
+ * @param dateSent - The date sent
+ * @returns The formatted date
+ * @example
+ * getFormattedDateSent('2024-01-01T12:30:00.000'); // '01 Jan 2024'
+ */
+export const getFormattedDateSent = (dateSent: IsoDateTimeStamp): string => format(parseISO(dateSent), SHORT_FORM_DATE);
+
+/**
+ * Maps completed corrections to a view model.
```

Comment:
more documentation here, especially for what correctionIdToDateSentDataSortValueMap  does